web/src/integration_url_modal.ts

```diff
@@ -242,10 +260,24 @@ export function show_generate_integration_url_modal(api_key: string): void {
             }
 
             const params = new URLSearchParams({api_key});
+            const selected_topic_option = topics_to_use_dropdown_widget.value();
             if (stream_id !== -1) {
                 params.set("stream", stream_id!.toString());
-                if ($override_topic.prop("checked") && topic_name !== "") {
+
+                // The topics_to_use_dropdown_widget is only used by the Slack integration,
```

Comment:
And once we rename the variables, we can drop this comment (just this line) too - "only used by the Slack integration".